vlib/v/fmt/tests/empty_lines_input.vv

```diff
@@ -0,0 +1,64 @@
+fn keep_single_empty_line() {
+	println('a')
+	println('b')
+
+	println('c')
+
+	d := 0
+
+	if true {
+		println('e')
+	}
+
+	f := 0
+}
+
+fn squash_multiple_empty_lines() {
+	println('a')
+
+
+	println('b')
+
+
+	c := 0
+
+
+	d := 0
+}
+
+fn remove_leading_and_trailing_empty_lines() {
+
+	println('a')
+
+	println('b')
+
+	if test {
+
+		c := 0
+
+	} else {
+
+		d := 0
+
+	}
+
+}
+
+fn no_empty_line_after_multi_line_statements() {
+	// line1
+	/* block1 
+	*/
+	/*
+	block2
+	*/
+	if test {
+		println('a')
+	}
+	println('b')
+	for test {
+		println('c')
+	}
+	c := fn(s string) {
+		println('s')
+	}
+}
```

Comment:
should this almost be same as the counterpart in _expected?

Reply:
yes, and I will move it to `_keep` as well. These were some cases where the `last_line` was set incorrectly and a newline was added after the statement.

Reply:
fixed